packages/cli/package.json

```diff
@@ -11,19 +11,19 @@
     "@hubspot/cli-lib": "^3.0.10-beta.9",
     "@hubspot/serverless-dev-runtime": "^3.0.10-beta.9",
     "archiver": "^5.3.0",
-    "chalk": "^4.1.0",
+    "chalk": "4.1.2",
     "express": "^4.17.1",
     "findup-sync": "^4.0.0",
     "fs-extra": "^8.1.0",
-    "inquirer": "^6.3.1",
+    "inquirer": "^8.2.0",
     "moment": "^2.29.1",
     "open": "^7.0.3",
     "ora": "^4.0.3",
     "shelljs": "0.8.3",
-    "spinnies": "^0.5.1",
+    "spinnies": "^0.3.2",
```

Comment:
In testing, I think this might cause an issue. I'll see if I can get to the bottom of it

Reply:
What was the issue? I tested it out a bit and it seemed to work for me.

Reply:
Getting `TypeError: spinnies.remove is not a function` when running `hs project upload`. Looks like the remove method was added in a later version